lib/nerves_runtime/init.ex

```diff
@@ -148,6 +148,9 @@ defmodule Nerves.Runtime.Init do
       {0, _} ->
```

Comment:
Isn't the problem that the tuple order is wrong here? I.e., it should be `{_, 0}`?